PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- Open this PR as draft while it is not ready -->
+
+### Description
+
+<!-- Describe your solution for the issue here. Adding videos, screenshots and expected logs may help the others to understand what is new.
+-->
+
+### Related Issues
+<!--
+Use keywords like 'close' or 'solves' to link this PR to an issue.
+For example:
+
+Closes #12345
+Unblocks #54321
+-->
+
+### Progress
+
+<!-- If your PR is WIP, use checkboxes to show that you did and what you have to do. For example: -->
+
+- [x] New endpoint created;
+- [ ] Update organizations;
+- [ ] Create tests;
+
+This PR solves #12345
+
+#### Pull request checklist
+
+<!-- Before submitting the PR, please address each item -->
+
+- [ ] **Tests**: This PR includes tests for covering the features or bug fixes, or it includes an explanation of why it does not.
```

Comment:
I'd prefer it as "(...)  This PR includes tests for covering the features or bug fixes (if applicable)."

I'm not sure about other quickstarts, but on the ones I know of one adds tests only occasionally (so that asking for an explanation everytime would be a little too much I think)

Reply:
Changed to not ask for a explanation why tests are missing.